gammapy/modeling/models/tests/test_time.py

```diff
@@ -58,3 +61,59 @@ def test_light_curve_evaluate_norm_at_time(light_curve):
 def test_light_curve_mean_norm_in_time_interval(light_curve):
     val = light_curve.mean_norm_in_time_interval(46300, 46301)
     assert_allclose(val, 0.021192284384617066)
+
+
+def rate(x, c="1e4 s"):
+    c = u.Quantity(c)
+    return np.exp(-x / c)
+
+
+def ph_curve(x, amplitude=0.5, x0=0.01):
+    return 1.0 + amplitude * np.sin(2 * np.pi * (x - x0) / 1.0)
+
+
+def test_time_sampling():
+    time = np.arange(0, 10, 0.06) * u.hour
+
+    table = Table()
+    table["TIME"] = time
+    table["NORM"] = rate(time)
+    temporal_model = LightCurveTableModel(table)
+
+    t_min = "2010-01-01T00:00:00"
+    t_max = "2010-01-01T08:00:00"
+
+    sampler = temporal_model.sample_time(
+        n_events=2, t_min=t_min, t_max=t_max, random_state=0, t_delta="10 min"
+    )
+
+    sampler_no_lc = LightCurveTableModel(None).sample_time(
```

Comment:
It think for this case we should just introduce a `TemporalConstant` time model. Please just remove the test case...

Reply:
Done.